src/rod_mechanics/softRobots.h

```diff
@@ -26,6 +26,9 @@ class softRobots
 
     void lockEdge(int limb_idx, int edge_idx);
     void applyInitialVelocities(int limb_idx, const vector<Vector3d>& velocities);
+    void applyPositionBC(const MatrixXd &positions);
+    void applyTwistBC(const MatrixXd &twists);
+    void applyCurvatureBC(const MatrixXd &Curvatures);
```

Comment:
The names for these is a bit weird for me.

I think a better name is `changeTwistBar` and `changeKappaBar`

Reply:
BC means boundary conditions

Reply:
Right, but we are not changing twist or curvature to a certain boundary condition value. Instead we are changing the natural twists / curvatures, which will indirectly result in a nonlinear change in twist or curvature. This is different from positionBC, which actually changes the position to the desired value.